design/reboot-interface.md

```diff
@@ -89,22 +89,34 @@ section of the BareMetalHost status. This records a time after which the Host
 was last booted using the current image. Processes running prior to this time
 may be assumed to have been stopped.
 
-A new date-time field, ``lastReboot``, will be added to the ``provisioning``
+A new date-time field, ``lastRebootRequest``, will be added to the ``provisioning``
```

Comment:
I'm actually quite partial to ``pendingRebootSince``, subject to confirmation that we're actually calling this a "reboot" (see comment below).